tlatools/org.lamport.tlatools/src/pcal/trans.java

```diff
@@ -1702,7 +1691,7 @@ else if (option.equals("-version") || (inFile && option.equals("version")))
                     {
                         throw new NumberFormatException();
                     }
-                    int a = new Integer(args[nextArg]).intValue();
+                    int a = Integer.valueOf(args[nextArg]);
```

Comment:
Nit: this call is wasteful because it allocates an `Integer` and then immediately unboxes it to `int`. It would be faster (and probably clearer) to use `Integer.parseInt` instead.

Reply:
Thanks for removing the deprecated call to `new Integer` though 👍